backend/recipes/__init__.py

```diff
@@ -55,14 +55,28 @@ def refresh():
     list = getList()
 
 
+def getRecipeByName(name):
+    """
+    Gets the full recipe object from it's name.
+    :param name:
+    The name of the recipe. This is it's title in the json file.
+    :return:
+    The recipe object or None if no recipe with given name could be found
+    """
+    global list
```

Comment:
Would it make sense to just recreate the list on each function call instead?  This way we can allow new recipes to be loaded without restarting the system, and the call itself is pretty cheap.  Also allows us to not use a global variable.

Reply:
Also feel free to tag me in future PRs if you want extra eyes on stuff

Reply:
Yeah that probably makes sense to do, it's already implicitly being called by listRecipes since that calls refresh before accessing the list.